src/OrbitGl/SourceCodeReport.cpp

```diff
@@ -29,6 +33,10 @@ SourceCodeReport::SourceCodeReport(std::string_view source_file,
     if (!maybe_current_line_info.has_value()) continue;
 
     const auto& current_line_info = maybe_current_line_info.value();
+
+    min_line_number_ = std::min(min_line_number_, current_line_info.source_line());
```

Comment:
Shouldn't these updates be before the continue??

Reply:
No the loop is not over line numbers! The loop is over instruction addresses. So we have to obtain the line number. I actually moved this even later.